### ISSUE TITLE
eslint component ignores --projenrc-ts flag

### ISSUE THREAD

**Issue by kadrach** [projen/projen#1135]
My projen-generated CDK app eslint task was misbehaving due to a known incompatibility between eslint@8 and the typescript plugin.

While fixing that, I noticed the following, minimal example below:

```bash
% npx projen new typescript --projenrc-ts
                      # Typescript ^

...
% npx projen eslint                                                                                                                                                                                                                                                                                                                                                                                                ~/tmp/foo main (v)
🤖 eslint | env: PATH=/Users/.../tmp/f...
🤖 eslint | eslint --ext .ts,.tsx --fix --no-error-on-unmatched-pattern src test build-tools .projenrc.js
                                                                                    # That's not right ^
```

The eslint component does not currently consider non-default projenrc types, it's either the default `.js` or nothing:

https://github.com/projen/projen/blob/fd76d2ece9fa6f3a3fa666b43c550913995e7ccf/src/eslint.ts#L137-L147

**Comment by kadrach** [comment]
Still present in `0.43.1`.

**Comment by marciocadev** [comment]
@Chriscbr I tested this issue and it works, I think some PR resolve this problem

You should close it

```colsole
$ projen new typescript --projenrc-ts

...

projen eslint
👾 eslint | eslint --ext .ts,.tsx --fix --no-error-on-unmatched-pattern src test build-tools projenrc .projenrc.ts
```

**Comment by mrgrain** [comment]
Agreed, this should be working now. I fixed some related issues in #1926

**Comment by kadrach** [comment]
I can no longer reproduce this issue with recent versions of the CDK. Thanks everyone!
